CorePluginFramework/src/au/gov/asd/tac/constellation/pluginframework/parameters/PluginParameter.java

```diff
@@ -70,6 +71,11 @@
     private boolean visible = true;
     private boolean enabled = true;
     private String helpID;
+    private boolean isSuppressed = false;
+    
+    private ArrayList<ParameterChange> suppressedEvents = new ArrayList<>();
+    
+    private static final ArrayList<ParameterChange> PARAMETER_EVENTS = new ArrayList<>(Arrays.asList(ParameterChange.values()));
```

Comment:
Ditto, can you please change to `private static final List`. It doesn't make too much difference, its more my OCD 😸 .